aad/spring-cloud-azure-starter-active-directory-b2c/aad-b2c-resource-server/README.md

```diff
@@ -1,13 +1,3 @@
----
-page_type: sample
-languages:
-- java
-products:
-- azure-active-directory-b2c
-description: "Sample project for Azure AD B2C Spring Boot resource server client library"
-urlFragment: "azure-spring-boot-sample-active-directory-b2c-resource-server"
----
```

Comment:
How about update the titles?
For example, from `Sample for Azure AD B2C Resource server Spring Boot client library for Java` to `spring-cloud-azure-starter-active-directory-b2c sample: Resource server`

Reply:
I deleted this headers in the Azure-Samples:spring-cloud-azure_4.0 branch